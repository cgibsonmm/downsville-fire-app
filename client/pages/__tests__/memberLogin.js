import { render, cleanup } from "@testing-library/react";
import MemberLogin from "../member-login";

beforeEach(cleanup);

describe("<MemberLogin />", () => {
  test("<MemberLogin />", () => {
    let { getByTestId, container } = render(<MemberLogin />);

    let usernameForm = getByTestId("usernameForm");
    let passwordForm = getByTestId("passwordForm");
    expect(passwordForm.textContent).toBe("");
    expect(usernameForm.textContent).toBe("");

    expect(container.firstChild).toMatchSnapshot();
  });
});
