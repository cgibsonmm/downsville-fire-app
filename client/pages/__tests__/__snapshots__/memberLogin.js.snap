// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MemberLogin /> <MemberLogin /> 1`] = `
<div
  class="w-screen p-4"
>
  <div
    class="flex flex-col w-full justify-center items-center bg-red-400 rounded-lg"
  >
    <h3
      class="text-xl"
    >
      Member Login
    </h3>
    <form
      class="flex flex-col w-full p-6"
      data-testid="loginForm"
    >
      <label
        for="email"
      >
        Email
      </label>
      <input
        class="bg-white focus:outline-none focus:shadow-outline border border-gray-300 rounded-lg py-2 px-4 block w-full appearance-none leading-normal"
        data-testid="usernameForm"
        name="email"
        type="email"
      />
      <label
        for="password"
      >
        Password
      </label>
      <input
        class="bg-white focus:outline-none focus:shadow-outline focus:text-xs border border-gray-300 rounded-lg py-2 px-4 block w-full appearance-none leading-normal"
        data-testid="passwordForm"
        name="password"
        type="password"
      />
      <button>
        Submit
      </button>
    </form>
  </div>
</div>
`;
